ui/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -5,22 +5,23 @@ import {Component} from '@angular/core';
 import {MatCardModule, MatTableModule} from '@angular/material';
 import {MatDividerModule} from '@angular/material/divider';
 import {RouterTestingModule} from '@angular/router/testing';
+import {ActivatedRoute, Router} from "@angular/router";
+import 'rxjs/add/observable/of';
 
 import {JobManagerService} from '../core/job-manager.service';
 import {SharedModule} from '../shared/shared.module';
-import {Router} from "@angular/router";
-import 'rxjs/add/observable/of';
 import {RouteReuse} from '../route-reuse.service';
 import {DashboardResolver} from "./dashboard.resolver.service";
 import {DashboardComponent} from "./dashboard.component";
 import {TotalSummaryComponent} from "./total-summary/total-summary.component";
 import {GroupedSummaryComponent} from "./grouped-summary/grouped-summary.component";
-import {FakeAggregationService} from "../testing/fake-aggregation.service";
+import {FakeAggregationService, CARD_NUM, TEST_AGGREGATION_RESPONSE} from "../testing/fake-aggregation.service";
 
 describe('DashboardComponent', () => {
   let testComponent: DashboardComponent;
   let fixture: ComponentFixture<AppComponent>;
   let fakeJobService: FakeAggregationService;
+  let testJobListComponent: TestJobListComponent;
```

Comment:
I don't think the variable makes sense here, since it is only populated within specific test cases. I would just remove this line and then put a `const` or `let` where you're defining it